python/src/GeneralLinearModelResult_doc.i.in

```diff
@@ -109,13 +109,13 @@ covModel : :class:`~openturns.CovarianceModel`
 
 // ---------------------------------------------------------------------
 
-%feature("docstring") OT::GeneralLinearModelResult::getBasisCollection
+%feature("docstring") OT::GeneralLinearModelResult::getBasis
 "Accessor to the collection of basis.
 
 Returns
 -------
-basisCollection : collection of :class:`~openturns.Basis`
-    Collection of the :math:`p` function basis: :math:`(\varphi_j^l: \Rset^d \rightarrow \Rset)_{1 \leq j \leq n_l}` for each :math:`l \in [1, p]`.
+basisCollection : :class:`~openturns.Basis`
```

Comment:
I think the type here is is wrong

Reply:
why ? It is a basis

Reply:
In the same way as for the krigingresult, shoudln't it be :  basis : :class:`~openturns.Basis` ?